homeassistant/components/tuneblade_remote/__init__.py

```diff
@@ -0,0 +1,52 @@
+"""Integration to integrate TuneBlade Remote devices with Home Assistant."""
+
+import logging
+
+from pytuneblade import TuneBladeApiClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import DOMAIN
+from .coordinator import TuneBladeDataUpdateCoordinator
+from .types import TuneBladeRuntimeData
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up TuneBlade integration from a config entry."""
+
+    host = entry.data[CONF_HOST]
+    port = entry.data[CONF_PORT]
+
+    session = async_create_clientsession(hass)
+
+    client = TuneBladeApiClient(
+        host=host,
+        port=port,
+        session=session,
+    )
+
+    coordinator = TuneBladeDataUpdateCoordinator(hass, client)
+
+    await coordinator.async_init()
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
+    entry.runtime_data = TuneBladeRuntimeData(coordinator=coordinator)
+
+    await hass.config_entries.async_forward_entry_setups(entry, ["media_player"])
```

Comment:
Please create a list PLATFORMS and use the `Platform` enum for this

Reply:
ok think I've done that correctly in const